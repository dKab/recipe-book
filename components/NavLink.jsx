import React from 'react';
import {Link} from 'react-router';

export const NavLink = (props) => {
    return (
        <Link activeClassName="active" {...props}  />
    );
};