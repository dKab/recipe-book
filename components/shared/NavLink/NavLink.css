.active {
  color: black !important;
  text-decoration: none;
  cursor: default;
}