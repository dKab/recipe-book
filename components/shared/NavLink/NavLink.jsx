import React from "react";
import { NavLink as Link } from "react-router-dom";
import styles from "./NavLink.css";

export const navLinkStyles = styles;

export const NavLink = props => {
  return (
    <Link className={styles.navLink} activeClassName="active" {...props} />
  );
};
