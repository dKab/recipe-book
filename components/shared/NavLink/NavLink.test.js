import React from 'react';
import {NavLink} from './NavLink.jsx'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(
      <NavLink to='/whatever'>Somewhere</NavLink>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});