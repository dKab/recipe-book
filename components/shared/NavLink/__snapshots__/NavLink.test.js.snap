// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<a
  className="navLink"
  onClick={[Function]}
  style={Object {}}
>
  Somewhere
</a>
`;
